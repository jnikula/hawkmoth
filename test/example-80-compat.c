/**
 * Compat comment transformations.
 *
 * Transformations require ``cautodoc_transformations`` configuration in
 * ``conf.py``. In this example, a transformation is used to support
 * Javadoc-style documentation comments.
 *
 * @param list The list to frob.
 * @param mode The frobnication mode.
 * @return 0 on success, non-zero error code on error.
 * @since v0.1
 */
int frob2(struct list *list, enum mode mode);
