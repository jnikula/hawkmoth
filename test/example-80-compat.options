{ "directive-options": { "transform": "javadoc-liberal" } }
