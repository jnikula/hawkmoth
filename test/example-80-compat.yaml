directive-arguments:
- example-80-compat.c
directive-options:
  transform: javadoc-liberal
example-title: Compat
example-priority: 80
