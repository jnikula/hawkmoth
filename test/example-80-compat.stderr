WARNING: 13: declaration of 'struct list' will not be visible outside of this function
WARNING: 13: declaration of 'enum mode' will not be visible outside of this function
